test/TestDummies/Console/DummyConsoleSink.cs

```diff
@@ -7,13 +7,13 @@ namespace TestDummies.Console
 {
     public class DummyConsoleSink : ILogEventSink
     {
-        public DummyConsoleSink(ConsoleTheme theme = null)
+        public DummyConsoleSink(ConsoleTheme? theme = null)
         {
             Theme = theme ?? ConsoleTheme.None;
         }
 
         [ThreadStatic]
-        public static ConsoleTheme Theme;
+        public static ConsoleTheme Theme = null!;
```

Comment:
I think a `null` value can be observed, here, if no sink has been constructed on the thread.

Reply:
fixed